README.md

```diff
@@ -65,6 +79,87 @@ To generate coverage results, set the `mapCoverage` property in the `jest` confi
 
 > Please note that the `outDir` property in the `jest` configuration section is removed in coverage mode, due to [#201](https://github.com/kulshekhar/ts-jest/issues/201).
 
+## Sensible Defaults
+`ts-jest` tries to ship with sensible defaults, to get you on your feet as quickly as possible.
+
+### Sourcemap support
+Sourcemaps should work out of the box. That means your stack traces should have the correct line numbers,
+and you should be able to step through the TypeScript code using a debugger.
```

Comment:
I'd like to get the debugger info in there, but I originally chose 'should' as the debugger doesn't work with all editors. Can we find a compromise?